### PR TITLE
FTB-123-Add Element Endpoints

### PR DESCRIPTION
**Pull request by josh-dadswell-hackney**
## Link to JIRA ticket

https://hackney.atlassian.net/browse/FBT-123

## Describe this PR

Adding endpoints for getting elements

### CODE REVIEW

File: BrokerageApi/appsettings.Development.json [hunk 1]
```diff
@@ -9,5 +9,5 @@
   "AWS": {
     "Region": "DefaultRegion"
   },
-  "DatabaseConnectionString": "Host=127.0.0.1;Port=5432;Username=postgres;Database=brokerage_api;Password=mypassword"
```
Comment (pixeltrix) [inline BrokerageApi/appsettings.Development.json:12]: Is this change intentional ?

**Review by pixeltrix — APPROVED** [review]
Just the question over the password change

File: BrokerageApi/appsettings.Development.json (hunk 1 above)
Comment (josh-dadswell-hackney) [reply BrokerageApi/appsettings.Development.json:12]: No had to add it to get my local DB connecting, is there s different way your doing it> Committed this by mistake

Comment (josh-dadswell-hackney) [reply BrokerageApi/appsettings.Development.json:12]: Oh ignore me I thought I had added it by mistake but infact I removed it by mistake
